src/components/UsageBarItem.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const UsageBarItemBody = styled.div`
+  display: flex;
+`;
```

Comment:
Does `flex` actually do anything for you if you just have text content?

Reply:
haha, nope! This had divs before and I realized I could the same with plain text 😆. good catch